src/main/java/Duke.java

```diff
@@ -9,9 +9,11 @@ public static void main(String[] args) {
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
 
-        String[] texts = new String[100];
+        Task[] texts = new Task[100];
         String line;
+        String secondline;
         int NumberOfTexts = 0;
+        int TextNumber;
```

Comment:
Perhaps these variables could be named in another way? You may want to double check the naming conventions for variables and classes/enums as it is quite similar.